leo/commands/gotoCommands.py

```diff
@@ -81,24 +83,25 @@ def find_node_start(self, p: Position, s: str = None) -> Optional[int]:
                 return i + 1
         return None
     #@+node:ekr.20150622140140.1: *3* goto.find_script_line
-    def find_script_line(self, n: int, root: Position) -> Tuple[Position, int, bool]:
+    def find_script_line(self, n: int, root: Position) -> Tuple[Position, int]:
         """
         Go to line n (zero based) of the script with the given root.
```

Comment:
Same here, This doc string should point out the return values for a failure to find the line is a (None, -1) tuple.